### DISCUSSION TITLE
No Database Issue

### DISCUSSION THREAD

**Discussion by cheezymousehouse825** [MusicPlayerDaemon/MPD#1365] [Q&A] [answered]
Hello, I am trying to get mpd working, however, when I try to access it via ncmpc it says in the bottom left "No Database" My config file is here: http://paste.debian.net/1221269/ and when I run mpd --verbose, all it says is loaded config file. I am using version 0.23.4 and I have done multiple things to try to get it to work. I have allowed port 6600 in my firewall, chowned my music directory to the mpd user,  and I have enabled mpd.service and mpd.socket with systemctl. Please help.

**Comment by cheezymousehouse825** [comment, marked as answer]
Solved by running `systemctl --user enable mpd.service` because before I ran it without the user flag, causing it to run as root instead of on my user account.

**Comment by Skrib** [comment]
I'm having this exact same issue, but on Artix with runit.
How can I run `systemctl --user enable mpd.service` without sysmctl? I can't find a similar command with loginctl.

**Reply by MaxKellermann** [reply]
You can't. systemctl is a front-end for systemd, and if you don't hav e systemd, you can't possibly have systemctl.

**Reply by Skrib** [reply]
I meant an equivalent to a 'systemctl enable' in runit. There's some options on the artix page for runit that look to have a similar purpose as systemctl, but nothing I've tried has had an effect on this ncmpcpp 'MPD: No database' error.

**Reply by cheezymousehouse825** [reply]
This: https://wiki.gentoo.org/wiki/MPD#Per-user_configuration should work for you. If not, try creating the db file manually.

**Reply by Skrib** [reply]
Thanks for the tips, but no luck unfortunately. I ended up reinstalling everything, and am no longer getting the No database error. However, I still cant load my library. I think my music folder is either too large or corrupted.
